### PR TITLE
feat: include npcs in townhall request

### LINKED ISSUE #25: feat: pass the list of the Realm's villagers on townhall request
When requesting a townhall through the WS in the UI, we need to pass a list of the villagers to our lore-machine

### PR DESCRIPTION
**Pull request by edisontim**
Uses the real NPCs that are in the Realm to pass that data to lore-machine when requesting a town hall meeting

Closes #25 

Needs #41 to be merged first

### CODE REVIEW

File: client/src/components/cityview/realm/villagers/NpcPanel.tsx [hunk 1]
```diff
@@ -85,9 +85,13 @@ export const NpcPanel = ({ type = "all" }: NpcPanelProps) => {
   }, [realmId]);
 
   const gatherVillagers = () => {
+    const npcsNoBigInt = npcs.map((npc): any => {
```
Comment (cwastche) [inline client/src/components/cityview/realm/villagers/NpcPanel.tsx:88]: We can prob find a better name for this var

Comment (edisontim) [reply client/src/components/cityview/realm/villagers/NpcPanel.tsx:88]: But I agree, I rushed that one

Comment (edisontim) [reply client/src/components/cityview/realm/villagers/NpcPanel.tsx]: Changed :)

**Review by cwastche — APPROVED** [review]
lgtm

**Pull request merged**